### PR TITLE
docs(ja): translate guides/configuration-glossary/configuration-generate

### PR DESCRIPTION
**Pull request by aytdm**
@inouetakuya @53able please review this PR.

#### Ref

- https://ja.nuxtjs.org/api/configuration-generate
- https://github.com/nuxt/nuxtjs.org/issues/544
- https://github.com/vuejs-jp/ja.nuxtjs.org/issues/22

### CODE REVIEW

File: content/ja/guides/configuration-glossary/configuration-generate.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+  }
+}
+```
+
+## concurrency
+
+- 型: `Number`
+- デフォルト: `500`
+
+ルート生成は並行で行われ、`generate.concurrency` は 1 スレッドで実行されるルートの量を指定します。
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-generate.md:64]: [IMO] amount を直訳して量にしたい気持ちも分かりつつ、ルートは数えられるので、ちょっと違和感がありました ☺️ 

```suggestion
ルート生成は並行で行われ、`generate.concurrency` は 1 スレッドで実行されるルートの数を指定します。
```

File: content/ja/guides/configuration-glossary/configuration-generate.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+export default {
+  generate: {
+    fallback: '404.html'
+  }
+}
+```
+
+フォールバックする HTML ファイルのパスです。エラーページとして設定する必要があります。この設定により、不明なルートも Nuxt を介してレンダリングされます。未設定またはファルシーな値が設定されている場合、フォールバック HTML ファイルの名前は `200.html` になります。もし、`true` を設定すると、ファイル名は `404.html` になります。値として文字列を指定すると、その文字列が代わりに使用されます。
+
+SPA を実行した場合は他のルートが生成されず必要はファイルは `200.html` のみとなるので、`200.html` を使うのが慣用的です。
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-generate.md:166]: [IMO] 語句だと慣用的という表現がピッタリくると思うのですが、振る舞いについては慣習的という表現のほうが、しっくり来ます。

https://eow.alc.co.jp/search?q=idiomatic

```suggestion
SPA を実行した場合は他のルートが生成されず必要はファイルは `200.html` のみとなるので、`200.html` を使うのが慣習的です。
```

ただ、あくまで IMO なのでお任せします 🙏

File: content/ja/guides/configuration-glossary/configuration-generate.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+```
+
+_注意: 複数のサービス（例えば Netlify）では、`404.html` を自動的に検出します。ウェブサーバーを独自に設定する場合は、ドキュメントを参照してエラーページの設定方法を確認してください（そして、エラーページを `404.html` ファイルに設定してください）_
+
+## interval
+
+- 型: `Number`
+- デフォルト: `0`
+
+2 つのレンダーの間でのインターバルで、ウェブアプリケーションからの潜在的な API に対して溢れでないようにするためのものです
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-generate.md:191]:
```suggestion
2 つのレンダーの間でのインターバルで、ウェブアプリケーションからの潜在的な API に対して溢れでないようにするためのものです。
```

**Review by inouetakuya — CHANGES_REQUESTED** [review]

**Comment by aytdm** [comment]
ありがとうございます、修正しました

**Review by inouetakuya — APPROVED** [review]
Thank you for your contribution :+1:

**Comment by debs-obrien** [comment]
@aytdm @inouetakuya I will let you guys go through all the ja prs and see if you can merge the ones you need to and if you come across any issues just let me know. So from now on I will only merge Japanese prs if for some reason you guys can't. Thanks for you fantastic work on contributing and reviewing and getting the new docs translated to Japanese. 😊

**Comment by aytdm** [comment]
Thank you. You've always been a great support 💚

**Comment by inouetakuya** [comment]
Thank you so much!! 💚

**Pull request merged**